apis/python/src/tiledbsoma/collection.py

```diff
@@ -190,6 +225,13 @@ def _add_new_ndarray(
         shape: Sequence[int],
         platform_config: Optional[options.PlatformConfig] = None,
     ) -> _NDArr:
+        """Adds a new NDArray to this Collection.
```

Comment:
Adds a new `SparseNDArray` or `DenseNDArray`

(These help-strings are user-facing and there is no user-facing `NDArray` class)

Reply:
I disagree and maintain my position. However I won't block the PR for this.

Reply:
I agree with Paul. IMHO, it is not confusing, and the brevity is appreciated.